src/themes/theme_jlmke.jl

```diff
@@ -0,0 +1,69 @@
+function theme_jlmke()
+    function jlmke_colors(alpha=1.0)
+        colors = [RGB(0.0, 0.0, 0.0), RGB(0.082, 0.643, 0.918), RGB(0.91, 0.122, 0.361),
+                  RGB(0.929, 0.773, 0.0), RGB(0.588, 0.196, 0.722), RGB(0.361, 0.722, 0.361),
+                  RGB(0.522, 0.522, 0.522)]
+        @. RGBAf(red(colors), green(colors), blue(colors), alpha)
+    end
```

Comment:
The function should probably live outside the main `theme_jlmke()` function, so that users can access the colors!

Reply:
good point.

Reply:
could you please to a suggestion change directly to the file? please.